course/steps/08_react-apollo/pt.md

```diff
@@ -0,0 +1,82 @@
+
+# Conectando *back-end* e *front-end*
```

Comment:
what do you think about standardazing how we write backend and frontend? 
- [ ] `back-end` and `front-end`
- [ ] `backend` and `frontend`